app/src/main/java/com/github/se/gatherspot/model/utils/EventUtils.kt

```diff
@@ -559,18 +559,47 @@ class EventUtils {
   }
 
   /**
-   * Check if an event is over
+   * Check if an event is over.
    *
    * @param event: The event to check
    * @return true if the event is over
    */
   fun isEventOver(event: Event): Boolean {
     val now = LocalDate.now()
-    val timeNow = LocalTime.now()
+    val timeNow = LocalTime.now().plusHours(2)
     return event.eventEndDate?.isBefore(now) == true ||
         (event.eventEndDate == now && event.timeEnding?.isBefore(timeNow) == true)
   }
 
+  /**
+   * Check if an event is underway.
+   *
+   * @param event: The event to check
+   * @return true if the event is underway
+   */
+  fun isEventUnderway(event: Event): Boolean {
+    val now = LocalDate.now()
+    val timeNow = LocalTime.now().plusHours(2)
+    val endDate: LocalDate = event.eventEndDate ?: event.eventStartDate!!.plusYears(10)
```

Comment:
we should change the default to be 2 day maximum. An event should not be open for 10 years

Reply:
fixed